server/hedley/modules/custom/hedley_reports/scripts/generate-well-child-completion-data.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * @file
+ * Generates completion data for different types of encounters.
+ *
+ * Execution: drush scr
+ *   profiles/hedley/modules/custom/hedley_reports/scripts/generate-well-child-completion-data.php.
+ */
+
+if (!drupal_is_cli()) {
+  // Prevent execution from browser.
+  return;
+}
+
+// Get the last node id.
+$nid = drush_get_option('nid', 0);
+
+// Get the number of nodes to be processed.
+$batch = drush_get_option('batch', 50);
+
+// Flag to generate data only if it was not generated already.
+$exclude_set = drush_get_option('exclude_set', FALSE);
+
+// Get allowed memory limit.
+$memory_limit = drush_get_option('memory_limit', 500);
+
+$type = 'individual_participant';
+$base_query = new EntityFieldQuery();
+$base_query
+  ->entityCondition('entity_type', 'node')
+  ->entityCondition('bundle', $type)
+  ->fieldCondition('field_encounter_type', 'value', 'well-child')
+  ->propertyCondition('status', NODE_PUBLISHED);
+
+$count_query = clone $base_query;
+$count_query->propertyCondition('nid', $nid, '>');
+$count = $count_query->count()->execute();
+
+if ($count == 0) {
+  drush_print("There are no nodes of type $type for well child encounters in DB.");
+  exit;
```

Comment:
```suggestion
  exit 1;
```

Is it kind of an error?